server/src/gas.ts

```diff
@@ -42,16 +42,15 @@ export class Gas {
         this.currentPosition = vClone(this.oldPosition);
     }
 
-    tick(): void {
+    tick(now: number, fwdMs: number): void {
         if (this.state !== GasState.Inactive) {
             this.percentage = (this.game.now - this.countdownStart) / (1000 * this.initialDuration);
             this.percentageDirty = true;
         }
 
-        this.ticksSinceLastDamage++;
         this.doDamage = false;
-        if (this.ticksSinceLastDamage >= 30) {
-            this.ticksSinceLastDamage = 0;
+        if ((now - this.lastDamageMs) > 1000) {
```

Comment:
I think previously it was counting 30 ticks, at 30ms per tick - about 1s per "do damage"?